docs/api-documentation.md

```diff
@@ -3431,16 +3431,17 @@ You must be a superuser to do this.
 
 ##### PARAMS:
 
-*  **`id`** 
+*  **`id`**
 
 ### `GET /api/user/`
 
 Fetch a list of `Users`. By default returns every active user but only active users.
 
-  If `status` is `deactivated`, include deactivated users only.
-  If `status` is `all`, include all users (active and inactive).
-  Also supports `include_deactivated`, which if true, is equivalent to `status=all`.
-  `status` and `included_deactivated` requires superuser permissions.
+  - If `status` is `deactivated`, include deactivated users only.
+  - If `status` is `all`, include all users (active and inactive).
+  - Also supports `include_deactivated`, which if true, is equivalent to `status=all`.
+  `status` and `include_deactivated` requires superuser permissions.
+  - `include_deactivated` is a legacy alias for `status` and will be removed in a future release, users are advised to use `status` for better support and flexibility.
```

Comment:
Here is the only part that changed, all other places are just remove trailing white space

Reply:
Oh, I should have mentioned -- you don't need to update this file. It's automatically generated from the code, usually once before each release.

Reply:
If there's a problem with the formatting, then please fix it in https://github.com/metabase/metabase/blob/master/src/metabase/cmd/endpoint_dox.clj
And then you can run `clojure -M:run api-documentation` to generate the docs.

But probably don't want to include the docs file in this PR, since it makes it more difficult to notice differences, when someone else runs the docs command.

Reply:
I reverted the changes in `docs/api-documentation.md` and fixed the trailing space in `endpoint_doc.clj`